src/Service/Database/ListTablesService.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Db\Migration\Service\Database;
+
+use Yiisoft\Db\Connection\Connection;
+use Yiisoft\Db\Exception\NotSupportedException;
+use Yiisoft\Yii\Console\ExitCode;
+use Yiisoft\Yii\Db\Migration\Helper\ConsoleHelper;
+use Yiisoft\Yii\Db\Migration\Service\MigrationService;
+
+use function array_column;
+use function array_merge;
+use function implode;
+use function preg_match;
+
+final class ListTablesService
```

Comment:
What's the use case for this command?

Reply:
That command allows listing the tables in the database, it is useful, when executing a migration I can see if I create them.